yuuto.js

```diff
@@ -9,7 +9,7 @@ dotenv.config(); // configures the environment variables
 export const prefix = process.env.PREFIX || "!";
 
 // Collections and Sets
-const yuuto = new Client();
+export const yuuto = new Client();
```

Comment:
There is absolutely no need for that
```suggestion
const yuuto = new Client();
```